paymentsheet/src/test/java/com/stripe/android/paymentsheet/FormPage.kt

```diff
@@ -37,4 +40,19 @@ internal class FormPage(
     private fun nodeWithLabel(label: String): SemanticsNodeInteraction {
         return composeTestRule.onNode(hasText(label))
     }
+
+    fun waitUntilVisible() {
+        composeTestRule.waitUntil(timeoutMillis = 2_000) {
```

Comment:
should we use the [DEFAULT_UI_TIMEOUT](https://github.com/stripe/stripe-android/blob/07bcd1c78581aebd1b1d02d6c91b47485044299f/paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/TestConstants.kt#L9) for this?

Reply:
We probably shouldn't specify anything here (I'll make sure that happens, either in this PR or a follow up).

We don't expect any delay for these (other than some animation transition time, and some async network response handling). Normally we specify a longer timeout for real network requests, but this is all happening in process.